rpc/json/rpc.json

```diff
@@ -242,24 +242,28 @@
         ]
     },
     {
-        "description": "Returns the information about a cell by out_point if it is live.",
+        "description": "Returns the information about a cell by out_point if it is live. If given second with_data argument, will return cell data and data_hash if it is live",
```

Comment:
where is the `data_hash` , I can't find it in result example.

Reply:
The `data` field is optional.
``` rust
pub struct CellInfo {
    pub output: CellOutput,
    pub data: Option<(JsonBytes, H256)>,
}
```

Reply:
I don't think that we need return datahash in rpc response, one data content field is enough

Reply:
It's just like the `hash` filed in `Transaction` struct, user do not need to calculate the hash themselves. If we attend to remove data_hash in `get_cell_data`, then I think remove hash here is reasonable.
https://github.com/nervosnetwork/ckb/blob/develop/rpc/json/rpc.json#L617

Reply:
Get, let's keep hash here. Please change tuple to named field struct for consistency:
```json
"data": {
    "content":  "0x...",
    "hash": "0x..."
}
```

Reply:
ok